content/blog/pkgdown-2-0-0/index.Rmd

```diff
@@ -0,0 +1,190 @@
+-   A number of minor improvements to accessibility, including a larger font size, greater use of `aria` attributes, and [accessible syntax highlighting colour scheme](https://apreshill.github.io/rmda11y/arrow.html), designed by Alison Hill.
+
+-   Support for new features like in-line footnotes and [tabsets in articles](https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html).
+
+-   Since so many packages have lovely logos, the package logo is now displayed on every page and made even more prominent on the home page.
+
+As an added incentive to upgrade your template to Bootstrap 5, you'll get site search for "free": pkgdown now supports searching with no external dependencies and no set-up.
+Learn more in `vignette("search")`.
+
+## Customisation
+
+The new template is also much easier to customise.
+A few of the most important features are noted below to whet your appetite; learn more in `vignette("customise")`.
+
+-   You can now easily change the overall visual appearance by picking a bootswatch theme:
+
+    ``` yaml
+    template:
+      bootstrap: 5
+      bootswatch: cyborg
+    ```
+
+    Or by selectively overriding the "bslib" variables used to generate the CSS:
+
+    ``` yaml
+    template:
+      bootstrap: 5
+      bslib:
+        bg: "#202123"
+        fg: "#B8BCC2"
+        primary: "#306cc9"
+        base_font: {google: "Roboto"}
+    ```
+
+    You can also choose a different syntax highlighting theme:
+
+    ``` yaml
+    template:
+      bootstrap: 5
+      theme: arrow-dark
+    ```
+
+-   You can now translate the English text that pkgdown contributes to each page.
+    This means that if you've written your package documentation in another language, you can ensure that language is also used on every part of the page.
+    Activate the translations by setting the `lang` field in `_pkgdown.yaml`, e.g.:
+
+    ``` yaml
+    lang: fr
+    ```
+
+    pkgdown includes translations for Spanish (es), German (de), French (fr), Portuguese (pt), Turkish (tr) and Chinese (zh_CN).
+    A big thanks to my colleagues who provided the initial translations, and to @dieghernan, @rivaquiroga, @jplecavalier who supplied additional improvements.
+    If you're interested in adding translations for your language please [file an issue](https://github.com/r-lib/pkgdown/issues) and we'll help you get started.
+
+-   You can add arbitrary HTML to every page with the new `includes` parameter.
+    This makes it easy to add analytics to your site, e.g. to use [plausible.io](https://plausible.io):
+
+    ``` yaml
+    templates:
+      includes:
+        in_header: |
+          <script defer data-domain="{YOUR DOMAIN}" src="https://plausible.io/js/plausible.js"></script>
+    ```
+
+-   The author, sidebar, and footer configuration is much more flexible allowing you to customise individual components while keeping most of the defaults (previously customisation was mostly all or nothing).
+    See `?build_home()` and `?build_site()` for details.
```

Comment:
not autolinked either

Reply:
Ok here it was because I had the `()` at the end